olca-app/src/org/openlca/app/editors/epds/UploadButton.java

```diff
@@ -42,8 +43,8 @@ void render(Composite parent, FormToolkit tk) {
 
 	private void update(Button btn) {
 		var text = epd().urn != null && epd().urn.startsWith("openEPD:")
-			? "Update EPD results on EC3"
-			: "Upload as draft to EC3";
+				? M.UpdateEc3
+				: M.UpdateEc3Draft;
```

Comment:
it is _upload_ not _update_ here, `UploadAsDraftToEC3` would be still a good key name (could be generated automatically from that string)